backend/beyond_the_loop/services/email_service.py

```
@@ -0,0 +1,188 @@
+from pathlib import Path
+from jinja2 import Environment, FileSystemLoader
+
+class EmailService:
+    def __init__(self):
+        self.configuration = sib_api_v3_sdk.Configuration()
+        self.configuration.api_key['api-key'] = os.getenv('BREVO_API_KEY')
+        self.api_instance = sib_api_v3_sdk.TransactionalEmailsApi(sib_api_v3_sdk.ApiClient(self.configuration))
+        
+        # Initialize Jinja2 environment
+        template_dir = Path(__file__).parent.parent / 'templates' / 'email'
+        self.jinja_env = Environment(loader=FileSystemLoader(template_dir))
```

Ruff (0.13.1)

By default, jinja2 sets `autoescape` to `False`. Consider using `autoescape=True` or the `select_autoescape` function to mitigate XSS vulnerabilities.

(S701)